### ISSUE TITLE
Review of configurations for UNESCO PROD

### ISSUE THREAD

**Issue by simboss** [geosolutions-it/unesco-ihp#342]
We shall do the usual complete review of the configuration for this GeoNode instance.

At the end of the review we should do a brief manual test of the GeoNode, comprehending a few uploads.
Make sure you clean what you uploaded.

**Comment by randomorder** [comment]
looks like there's no watchdog either. Take the change to set it up and review the monitoring setup.

**Comment by lpasquali** [comment]
> looks like there's no watchdog either. Take the change to set it up and review the monitoring setup.
> 
> http://ihp-wins.unesco.org/

@randomorder since it is the first time I add watchdogs:

1) where should I pick as example?
2) should I make a wiki page about configuring them?

**Comment by randomorder** [comment]
> 1. where should I pick as example?

For inspiration:
https://docs.google.com/document/d/1__PAiacn3_5iSEOWnkZ8fBMQnjt9KgKqMPAhTKT2dIE/edit#bookmark=id.44sucmcu1wt
https://docs.google.com/document/d/1PAcisNox6LwWEDkaEdT9LbTBKyht3M_ERgdAg3v2gLQ/edit#bookmark=id.mf0kpfjn681
Try to improve where possible keeping in mind the experience gained from GCH setup

> 2. should I make a wiki page about configuring them?

Yes please

**Comment by lpasquali** [comment]
@randomorder I configured watchdogs, going to make wiki

**Comment by lpasquali** [comment]
@randomorder could you review the assessment done and decide what to change?

**Comment by randomorder** [comment]
@agpenton is working on the watchdog for geoserver

**Comment by lpasquali** [comment]
@randomorder I produced and shared with you a document on the day I put thins on review, I'm veery sorry I missed to paste it here:
https://docs.google.com/document/d/14SVgPwGy-Jl7ZdybsELxr0eYUU1YHNLq-Jd-X5G1kdE/edit#heading=h.3od59fcx5ycw

**Comment by lpasquali** [comment]
```
Log rotation is configured
Layer Upload test done (success)
No Captcha/Recaptcha present (google api key is missing in /home/geosolutions/work/repo/ihp/ihp/local_settings.py
GeoNode admin password is not written anywhere, I logged in as alpa / jje9fZ9s
Databases password is default geonode/geonode
Sudo is passwordless (it should not) 
Disk quota is not on JNDI
```

**Comment by lpasquali** [comment]
concerning watchdog I enabled a watchdog takking the examples you gave me for geonode installations, there was as typo preventing the stop which I fixed job here:

`/usr/local/bin/geoserver-watchdog.sh`

**Comment by randomorder** [comment]
> @agpenton is working on the watchdog for geoserver

I said @agpenton was working on it though...

Have you tested it this time @lpasquali ?

**Comment by lpasquali** [comment]
> > @agpenton is working on the watchdog for geoserver
> 
> I said @agpenton was working on it though...
> 
> Have you tested it this time @lpasquali ?

yes it should work correctly, but if I can stop for 1 minute geoserver that would for certain

**Comment by randomorder** [comment]
please sync with @giohappy in pvt to test it

**Comment by lpasquali** [comment]
@giohappy , @randomorder tested, it worked, I stopped geoserver and watchdog restarted it

**Comment by randomorder** [comment]
thanks

**Comment by randomorder** [comment]
> @randomorder I produced and shared with you a document on the day I put thins on review, I'm veery sorry I missed to paste it here:
https://docs.google.com/document/d/14SVgPwGy-Jl7ZdybsELxr0eYUU1YHNLq-Jd-X5G1kdE/edit#heading=h.3od59fcx5ycw

I've added a few comments but to be honest it's shallow and not written @lpasquali 

Let's have a quick call tomorrow morning and re-review this together

**Comment by lpasquali** [comment]
@randomorder https://docs.google.com/document/d/1rj5tHCLlthCprn1NDRjDVM4Kguz19IG1oH0j9Dbew7A/edit#

**Comment by simboss** [comment]
can we close this @lpasquali @randomorder ?

**Comment by lpasquali** [comment]
@simboss on my side I'd said yes, but since is first application of the redaction of a new template for reviewing GeoNode installations I'd wait for @randomorder opinion
